openassessment/templates/openassessmentblock/staff_area/oa_staff_override_assessment.html

```diff
@@ -3,25 +3,36 @@
 
     <div class="wrapper--staff-assessment">
         <div class="step__instruction">
-            <p>{% trans "Override this learner's current grade using the problem's rubric." %}</p>
+            {% if not is_team_assignment %}
+                <p>{% trans "Override this learner's current grade using the problem's rubric." %}</p>
+            {% else %}
+                <p>{% trans "Override this team's current grade using the problem's rubric." %}</p>
+            {% endif %}
         </div>
 
         <div class="step__content">
             <article class="staff-assessment">
                 <div class="staff-assessment__display">
                     <header class="staff-assessment__display__header">
                         <h3 class="staff-assessment__display__title">
-                            {% if student_username %}
-                                {% blocktrans %}
-                                    Response for: {{ student_username }}
-                                {% endblocktrans %}
+                            {% if not is_team_assignment %}
+                                {% if student_username %}
+                                    {% blocktrans %}
+                                        Response for: {{ student_username }}
```

Comment:
Are we only showing the "Response for:" block for individuals? There's code in the staff management tamplate that gets the team name if we want to show that.

Reply:
The ticket didn't mention showing team name - just replacing learner with team. Perhaps a follow up clean up ticket?